packages/__examples__/.storybook/stories/stories.ts

```diff
@@ -109,11 +156,24 @@ console.log(
   }
 )
 
-// Stories not tied to one component
-storiesOf('Tooltip positioning', module).add('Tooltip positions', () =>
-  TooltipPositioning()
-)
-numStories++
+additionalExamples.forEach((example) => {
+  const { title, stories } = example
+
+  const storiesOfExample = storiesOf(title, module)
+  stories.forEach((story) => {
+    storiesOfExample.add(story.storyName, story.storyFn, {
+      chromatic: chromaticSettings,
+      ...(story.chromaticSettings || {})
+    })
```

Comment:
here you added `story.chromaticSettings` in the wrong place

Reply:
Thanks, fixed :)